scripts/skills/perks/perk_rf_discovered_talent.nut

```diff
@@ -30,22 +31,27 @@ this.perk_rf_discovered_talent <- ::inherit("scripts/skills/skill", {
 		::Math.seedRandom(1000 * actor.getUID() + ::toHash(this.getID()) + this.m.AttributesRolled.len() * 10000);
 
 		local choice = ::MSU.Array.rand(potential);
-		local toAdd = ::MSU.Class.WeightedContainer([
-			[70, 1],
-			[20, 2],
-			[10, 3]
-		]).roll();
 
 		::Math.seedRandom(::Time.getRealTime());
 
-		actor.getTalents()[choice] = ::Math.min(3, actor.getTalents()[choice] + toAdd);
+		actor.getTalents()[choice] = ::Math.min(this.m.MaxStars, actor.getTalents()[choice] + this.rollStars(choice));
```

Comment:
the rolling of the stars should also be seeded, therefor moved above the reseeding in line 35